htdocs/web_portal/controllers/downtime/view_endpoint_tree.php

```diff
@@ -19,31 +19,34 @@
  * limitations under the License.
  *
  /*====================================================== */
-function getServiceandEndpointList() {
+ 
+ // This is to authenticate request, lookup a User object by user's principle ID  and decides user is an Admin or NOT.
+function initAuthRequest() {
     require_once __DIR__ . '/../utils.php';
     require_once __DIR__ . '/../../../web_portal/components/Get_User_Principle.php';
 
     $dn = Get_User_Principle();
     $user = \Factory::getUserService()->getUserByPrinciple($dn);
     $params['portalIsReadOnly'] = portalIsReadOnlyAndUserIsNotAdmin($user);
+}
+
+function getServiceandEndpointList() {
```

Comment:
should the "and" be camel cased, to be getServiceAndEndpointList()? would need to be changed elsewhere when it is called

Reply:
Updated. I think it is carry forwarded.  Nice catch